contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV0.sol

```diff
@@ -277,15 +301,20 @@ contract MinterMerkleV0 is ReentrancyGuard, IFilteredMinterMerkleV0 {
         );
 
         // limit mints per address by project
-        if (projectMintedBy[_projectId][msg.sender]) {
-            require(
-                projectMintLimiterDisabled[_projectId],
-                "Limit 1 mint per address"
-            );
-        } else {
-            // EFFECTS
-            projectMintedBy[_projectId][msg.sender] = true;
-        }
+        uint256 _maxProjectInvocationsPerAddress = projectUseMaxInvocationsPerAddressOverride[
+                _projectId
+            ]
+                ? projectMaxInvocationsPerAddressOverride[_projectId]
+                : DEFAULT_MAX_INVOCATIONS_PER_ADDRESS;
+
+        // FINAL CHECK, WITH FOLLOW-ON EFFECTS
+        require(
+            // _++ returns current invocations, then increments
+            projectUserInvocations[_projectId][msg.sender]++ <
```

Comment:
Nit: Does doing incrementing and reading in this way having meaningful gas savings associated with it? I find this more confusing to read than separating out the ops re: order of operations here and wonder if it is worth keeping more readable.

Reply:
might be some gas savings here, but very small. broke out for improved readability, and also put increment in unchecked block, with comment `// this will never overflow since user's invocations on a project are limited by the project's max invocations`, which should be a net-reduction in mint gas cost.